src/flintxx/matrix.h

```diff
@@ -21,7 +21,7 @@
 #include "ltuple.h"
 #include "traits.h"
 #include "tuple.h"
-#include "permxx.h"
+#include "../permxx.h"
```

Comment:
I'm assuming here that as before, the files from `src/flint_public` should be installed to `/usr/include/flint` and the files from `src/flintxx` to `/usr/include/flint/flintxx`.
(The installation method described in the INSTALL file doesn't really work.)

Reply:
That is also partially my assumption, and I indeed plan to move the files from `src/flint_public` to `src`.

However, I would use system includes for flint headers, as I can see, `flint` being installed in `/usr/include/flint/` and `flintxx` being installed in `/usr/local/include/flint/`

Reply:
@fredrik-johansson what do you think?